nevergrad/parametrization/core.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, **parameters: tp.Any) -> None:
         # Additional convenient features
         self._random_state: tp.Optional[np.random.RandomState] = None  # lazy initialization
         self._generation = 0
-        self._constraint_checkers: tp.List[tp.Callable[[tp.Any], bool]] = []
+        self._constraint_checkers: tp.List[tp.Union[tp.Callable[[tp.Any], bool], tp.Callable[[tp.Any], float]]] = []
```

Comment:
would that work? `tp.List[tp.Callable[[tp.Any], tp.Union[bool, float]]]`
not sure though, but that would simplify the code

Reply:
Works!